tests/aeroval/test_setup_classes.py

```diff
@@ -4,7 +4,7 @@
 
 import pytest
 
-from pyaerocom.aeroval.setupclasses import EvalSetup
+from pyaerocom.aeroval import EvalSetup
```

Comment:
:-)

Reply:
Whoops, sorry, totally messed that up last time. Should be fixed now!